cg/resources/__init__.py

```diff
@@ -8,6 +8,12 @@
 
 RNAFUSION_BUNDLE_FILENAMES: str = Path("resources", "rnafusion_bundle_filenames.csv").as_posix()
 
+TAXPROFILER_BUNDLE_FILENAMES: str = Path("resources", "taxprofiler_bundle_filenames.csv").as_posix()
```

Comment:
Out of scope, but these should be made into an enum class and defined in constants.

Reply:
Making an issue for that.